### PR TITLE
BF: Set the `Diso` parameter value to the one provided 

### PR DESCRIPTION
**Pull request by jhlegarreta**
Set the `Diso` parameter value to the one provided in `wls_fit_tensor`.

### CODE REVIEW

**Comment by jhlegarreta** [comment]
Have not investigated why that was there.

**Comment by samcoveney** [comment]
This might be there reasons related to instability of the numerics - I have found that constrained DKI fails if you don't scale the units, and this might have been a setup to fixing that problem more generally, but I am not sure...

File: dipy/reconst/fwdti.py [hunk 1]
```diff
@@ -344,7 +344,7 @@ def wls_iter(
 
 
 def wls_fit_tensor(
-    gtab, data, Diso=3e-3, mask=None, min_signal=1.0e-6, piterations=3, mdreg=2.7e-3
+    gtab, data, mask=None, min_signal=1.0e-6, piterations=3, mdreg=2.7e-3
```
Comment (skoudoro) [inline dipy/reconst/fwdti.py:347]: Hi @jhlegarreta,

Diso should not be removed but it should be used here:

https://github.com/dipy/dipy/blob/8be51800309373135cdda26bbb7bfd9d310c1d41/dipy/reconst/fwdti.py#L423

Can you update this, then it is ready to go. Thanks!

**Review by skoudoro — CHANGES_REQUESTED** [review]
Hi @jhlegarreta,

See my comment below. Can you update this when you have time. Thanks!

File: dipy/reconst/fwdti.py (hunk 1 above)
Comment (jhlegarreta) [reply dipy/reconst/fwdti.py]: :+1: Nice catch. Done.

**Review by skoudoro — APPROVED** [review]
Thank you for the update @jhlegarreta , merging !

**Pull request merged**